app/controllers/catalog_controller.rb

```diff
@@ -62,6 +46,18 @@ def show
     end
   end
 
+  def branding
```

Comment:
Minor: Would throw this under a `private` so it doesn't get confused as a controller "action"

Reply:
Good idea!  I've made that change and fixed some rubocop nits about this class at the same time.